template.moonxml

```diff
@@ -28,6 +28,12 @@ ROOT = ->
 --           span "]:"
 --       else
 --         div class: "kv code-#{type @}", inspect @
+stack = {
+  push: table.insert
+  pop: table.remove
+  find: (target) => for index, item in ipairs @
```

Comment:
Quick alignment fix, because I'm a nitpicker.
```suggestion
  push: table.insert
  pop:  table.remove
  find: (target) => for index, item in ipairs @
```